### ISSUE TITLE
Sensorbezeichnung anpassen? 

### ISSUE THREAD

**Issue by GrandAmmor** [martinknechtel/sungrow-tibber-home-assistant#1]
Nach dem Kopieren und Einfügen der yaml-Inhalte gem. Anleitung muss noch ggf. die Sensorbezeichnung angepasst werden. So ist bei mir der sensor.tibber_electricity_price nicht vorhanden und heißt dafür sensor.electricity_price_home.

Vlt. lässt sich dazu noch ein Hilfssensor in die Automation einbauen, der den sensor.tibber_* referenziert und man den Quellsensor nur einmal an anderer Stelle anpassen muss. 

Danch funktioniert die Test-Automation bisher super.

**Comment by martinknechtel** [comment]
Danke @GrandAmmor ! Du meinst sozusagen als Alias, so dass ich nicht Suchen&Ersetzen für alle Treffer im yaml file machen muss sondern nur an einer Stelle Suchen&Ersetzen mache, also den Wert ```sensor.tibber_electricity_price``` ersetze so dass er zu meiner lokalen Installation passt, zum Beispiel auf ```sensor.martinshometibber_electricity_price```. Meinst du das so?

**Comment by GrandAmmor** [comment]
Ja, genau. Das ist zwar schon wirklich für faule Anwender, kann dem ein oder anderen aber sicher helfen, wenn man nur an einer Stelle 'seinen eigenen' Sensor eintragen muss.

Ich bekomme nun seit zwei Tagen stündlich Benachrichtigungen, hatte aber noch keinen 'Einkaufsfall'. Ich bin gespannt :)

**Comment by martinknechtel** [comment]
Hi @GrandAmmor funktioniert es bei dir wie es soll? Heute Abend fällt der Tibber Preis wieder ;-)

Ich mache den Thread hier erstmal zu, das verringerte copy&paste verringert die Komplexität, der zusätzliche Hilfssensor würde sie allerdings erhöhen. Vielleicht finden wir noch eine andere Idee.

**Comment by GrandAmmor** [comment]
Guten Morgen @martinknechtel,

bisher hatte ich noch keine 'Einkaufsbedingungen' und bin gespannt!

Mir ist zwischendurch bei blauem Himmel aufgefallen, dass 'bypassing battery' dazu führte, dass überschüssiger PV-Strom ins Grid eingespeist und nicht in den Akku geladen wurde.

Das konnte ich leider nicht systematisch nachvollziehen, da ich nur zufällig unterwegs darauf aufmerksam wurde. Das werde ich bei Gelegenheit aber noch mal beobachten.

Frohes neues Jahr! :)

**Comment by martinknechtel** [comment]
Dir auch ein frohes neues Jahr! :-) Ja genau, hast Recht, das fiel mir auch auf, deshalb habe ich `exporting power` noch als Trigger eingefügt, siehe https://github.com/martinknechtel/sungrow-tibber-home-assistant?tab=readme-ov-file#add-further-conditions-to-refine. Wenn man das nicht macht und der Strompreis ist gerade hoch, passiert genau das was du auch schon beobachtet hast.
